### PR TITLE
Bring release instructions up to date and inline test plan

### PR DESCRIPTION
**Pull request by cocreature**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-assistant/daml-helper/src/DamlHelper/Run.hs [hunk 1]
```diff
@@ -659,9 +659,9 @@ runStart (StartNavigator shouldStartNavigator) (OpenBrowser shouldOpenBrowser) o
     mbScenario :: Maybe String <-
         requiredE "Failed to parse scenario" $
         queryProjectConfig ["scenario"] projectConfig
-    let darPath = "dist" </> projectName <> ".dar"
+    let darPath = ".daml" </> "dist" </> projectName <> ".dar"
```
Comment (afsofia) [inline daml-assistant/daml-helper/src/DamlHelper/Run.hs]: ... good thing we have a manual test plan :grimacing:

File: release/RELEASE.md [hunk 2]
```diff
@@ -4,28 +4,62 @@
...
+   new version is visible.
+1. Run through the following test plan on Linux or MacOS:
 
-   The test plan currently still targets the old `da` assistant. We
-   will migrate the test plan fully to the new assistant soon but for
-   now, there is a shorter test plan for the new assistant that
-   you should run on Windows:
+   1. Install the SDK using `curl -sSL https://get.daml.com/ | sh -s $SDK_VERSION`.
+   1. Run `daml version --assistant=yes` and verify that `$SDK_VERSION` is the 
+      assistant version and the default version for new projects.
```
Comment (associahedron) [inline release/RELEASE.md]: yes! 👏

File: release/RELEASE.md [hunk 3]
```diff
@@ -4,28 +4,62 @@
...
+   1. Create a new project with `daml new quickstart quickstart-java`
+      and switch to it using `cd quickstart`.
+   1. Run `daml start`. Your browser should be opened automatically at
+      `http://localhost:7500`. Login as `Alice` and verify that there is
+      1 contract and 3 templates. Close the tab and kill `daml
+      start` using `Ctrl-C`.
+   1. Run `daml build -o .daml/dist/quickstart.dar`.
```
Comment (associahedron) [inline release/RELEASE.md]: Is the `-o .daml/dist/quickstart.dar` necessary here, given that this is the default location?

Comment (cocreature) [reply release/RELEASE.md]: Good point, removed.

**Pull request merged**